### PR TITLE
Setup shared-ui library for common ui components

### PR DESCRIPTION
**Pull request by storyworks**
I'm thinking we should house Text / Icon / Button in a common lib?

Keeping this basic so it can be quickly merged / John and I can use as a base for Text / Icon

This is literally just using the NX generator to create a new lib + giving it a basic component to test it works with storybook

### CODE REVIEW

File: workspace.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
           }
         },
         "serve-all": {
-          "builder": "@nrwl/workspace:run-commands",
+          "executor": "@nrwl/workspace:run-commands",
```
Comment (storyworks) [inline workspace.json:38]: I'm not sure why this changed - it occurred after I used nx generate to create shared-lib

Comment (storyworks) [reply workspace.json:38]: Spoke to Tatai, executor is the one we want.

**Comment by storyworks** [comment]
Bump? @tataihono , @gdub01 , @GeronimoJohn

**Review by GeronimoJohn — APPROVED** [review]
Looks good!

**Pull request merged**